### DISCUSSION TITLE
initialScrollOffset support

### DISCUSSION THREAD

**Discussion by DaveLomber** [Shopify/flash-list#700] [Q&A]
There is https://shopify.github.io/flash-list/docs/usage#initialscrollindex `initialscrollindex` supported.

Is there any way to have/do  `initialScrollOffset` ?

**Comment by khushal87** [comment]
Hey @naqvitalha, can we also set `viewPosition` and `animated` somehow? If I would like re-center my view after scroll to first index?

**Comment by Freddy03h** [comment]
Hi!

Since v2, `initialScrollOffset` doesn't work anymore because flash-list is not based on RecyclerListView anymore.

We've got `initialScrollIndex` and `initialScrollIndexParams` now, but how to re-create those values from the `offset` we receive in the `onScroll` ?

Thank's
